src/chapters/configuring.xml

```diff
@@ -6002,12 +6002,13 @@ guaclog: INFO: All files interpreted successfully.</computeroutput>
                 connection configuration itself, so long as the remote desktop connection uses the
                 same credentials as Guacamole.</para>
             <para>Each token is of the form
-                    <varname>${<replaceable>TOKEN_NAME</replaceable>}</varname>, where
-                    <replaceable>TOKEN_NAME</replaceable> is some descriptive name for the value the
-                token represents. Tokens with no corresponding value will never be replaced, but
-                should you need such text within your connection parameters, and wish to guarantee
-                that this text will not be replaced with a token value, you can escape the token by
-                adding an additional leading "$", as in "$${TOKEN_NAME}".</para>
+                    <varname>${<replaceable>TOKEN_NAME</replaceable>:<replaceable>MODIFIER</replaceable>}</varname>,
```

Comment:
I think this should be clarified to document the form as "`${TOKEN_NAME}` or `${TOKEN_NAME:MODIFIER}`", to avoid giving the impression that a modifier is required.

Reply:
Yeah, good point. I've modified it to include some language that should clarify that.